src/Illuminate/Auth/Access/Gate.php

```diff
@@ -100,20 +100,25 @@ public function has($ability)
      *
      * @param  string  $ability
      * @param  callable|string  $callback
+     * @param  array  $options
      * @return $this
      *
      * @throws \InvalidArgumentException
      */
-    public function define($ability, $callback)
+    public function define($ability, $callback, array $options = [])
     {
+        $options += ['guestable' => false];
```

Comment:
this is a bad way to merge arrays. You're better off keeping a protected property on the gate containing default options and doing an `array_merge`.